usr/sbin/p11sak/p11sak.c

```diff
@@ -1173,15 +1351,22 @@ static CK_RV sec_key_print_attributes(CK_SESSION_HANDLE session,
     CK_BBOOL a_never_extractable;
     CK_ULONG bs = sizeof(CK_BBOOL);
 
-    CK_ATTRIBUTE bool_tmplt[] = { { CKA_TOKEN, &a_token, bs }, { CKA_PRIVATE,
-            &a_private, bs }, { CKA_MODIFIABLE, &a_modifiable, bs }, {
-    CKA_DERIVE, &a_derive, bs }, { CKA_LOCAL, &a_local, bs }, {
-    CKA_SENSITIVE, &a_sensitive, bs }, { CKA_ENCRYPT, &a_encrypt, bs }, {
-            CKA_DECRYPT, &a_decrypt, bs }, { CKA_SIGN, &a_sign, bs }, {
-    CKA_VERIFY, &a_verify, bs }, { CKA_WRAP, &a_wrap, bs }, {
-    CKA_UNWRAP, &a_unwrap, bs }, { CKA_EXTRACTABLE, &a_extractable, bs }, {
-            CKA_ALWAYS_SENSITIVE, &a_always_sensitive, bs }, {
-            CKA_NEVER_EXTRACTABLE, &a_never_extractable, bs }, };
+    CK_ATTRIBUTE bool_tmplt[] = { 
+        { CKA_TOKEN, &a_token, bs }, 
```

Comment:
In general it would be saver to use sizeof() with the variable used as data:
`{ CKA_TOKEN, &a_token, sizeof(a_token) },`
But since those are all CK_BBOOLs (at least currently), it works like this, too.

Reply:
Done